Gemfile.lock

```diff
@@ -348,4 +348,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   1.13.6
+   1.14.5
```

Comment:
Doing so encourages future committers to keep their local bundler up to date, which seems reasonable. Assuming we don't just merge changes to a lower version in further PRs, that is. WDYT?